### PR TITLE
Integrated SonarCloud into Travis-CI

### PR DESCRIPTION
**Pull request by RJDiCosmo**
Added an addon and set of scripts for Travis-CI to use SonarCloud via the .travis.yml file.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,6 +1,19 @@
 language: java
 jdk: 
   - openjdk8
+addons:
+  # https://docs.travis-ci.com/user/sonarcloud/
+  sonarcloud:
+    organization: "revaturelabs-screenforce" # the key of the org
+    token:
+      secure: "*********" # encrypted value of your token
```
Comment (Quinn-Donnelly) [inline .travis.yml]: Can we change this to an envirment variable called `sonar-cloud-token`

File: .travis.yml [hunk 2]
```diff
@@ -1,6 +1,19 @@
...
+  depth: false
 script:
   - mvn install -DskipTests=true -Dmaven.javadoc.skip=true -B -V
-  - mvn test -B
\ No newline at end of file
+  - mvn test -B
+  # SonarQube/SonarCloud - other script steps might be done before running the actual analysis
+  - sonar-scanner
+  # the following command line builds the project, runs the tests with coverage and then execute the SonarCloud analysis
+  - mvn clean verify sonar:sonar -Pcoverage -Dsonar.projectKey=myorg_myrepo
```
Comment (Quinn-Donnelly) [inline .travis.yml]: Project Key: screenforce:tech-screening-service
Org Key: revaturelabs-screenforce

**Review by Quinn-Donnelly — CHANGES_REQUESTED** [review]
If we can make the few changes recommended in the comment I will move forward with the merge.

File: .travis.yml (hunk 1 above)
Comment (Quinn-Donnelly) [reply .travis.yml]: Also to clarify I am talking the sonarcloud.token.secure

**Review by erchiggins — APPROVED** [review]

**Pull request merged**